src/web.ts

```diff
@@ -1,22 +1,9 @@
-const enc = new TextEncoder();
-
-export async function sign(secret: string, data: string) {
-  const signature = await crypto.subtle.sign(
-    "HMAC",
-    await importKey(secret),
-    enc.encode(data)
-  );
-  return UInt8ArrayToHex(signature);
-}
+// @ts-ignore: ts(2691) for deno compatible
```

Comment:
Is this absolutely necessary?

We don't use `ts-ignore` since it can hide problems, rather we use `ts-expect-error`

Reply:
it seem that this will cause build failed.

but if do not add the `.ts` extension, deno wouldn't be able to run.

Reply:
Part of the problem is that we don't publish typescript, we publish JS, so ideally deno should be using the build output and not the TS directly

Reply:
```suggestion
// @ts-expect-error: ts(2691) for deno compatibility
```

Reply:
You are right, let deno just use the build output. Everything works well.